CHANGELOG.rst

```diff
@@ -5,7 +5,100 @@ Changelog
 TBR
 ---
 
-* Official support for Python 3.11
+This release enables scrapy-poet to fully support item types as dependencies in
```

Comment:
I think we should normalize “item type” / “item class” usage, i.e. choose one and stick to it (there is an “item class” reference a bit later in these changes to the changelog). I have a slight preference for the latter, but I am OK with either.

Reply:
You're right. The same is true for **web-poet** and we'll need to update it as well.

My preference is with "item class" as it's more straightforward of the idea that user-defined classes are used. On the other hand, "item type" could portray the idea of type annotations (e.g. `Union[ProductItem, ReviewItem]`) that _might_ create some confusion.

@kmike @wRAR, what are your thoughts?

Reply:
+1 to use "class". Reasoning: https://github.com/scrapinghub/web-poet/pull/110#discussion_r1047496239

Reply:
Updated to "item class" after our off-GH discussion.